common/src/main/java/org/opensearch/ml/common/connector/functions/preprocess/ConnectorPreProcessFunction.java

```diff
@@ -46,6 +47,10 @@ public void validateTextDocsInput(MLInput mlInput) {
         if (!(mlInput.getInputDataset() instanceof TextDocsInputDataSet)) {
             throw new IllegalArgumentException("This pre_process_function can only support TextDocsInputDataSet");
```

Comment:
If you have context can you please add class and method level comments, somehow that is missing for this class.

Reply:
Done